### PR TITLE
bump luajit

### PR DESCRIPTION
**Pull request by ElvinEfendi**
rel https://github.com/Shopify/luajit-build/pull/8

@Shopify/edgescale

### CODE REVIEW

File: luajit-shopify.rb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 class LuajitShopify < Formula
   desc "Just-In-Time Compiler (JIT) for the Lua programming language"
   homepage "http://luajit.org/luajit.html"
-  url "http://luajit.org/download/LuaJIT-2.1.0-beta2.tar.gz"
+  url "http://luajit.org/download/LuaJIT-2.1.0-beta3.tar.gz"
   sha256 "713924ca034b9d99c84a0b7b701794c359ffb54f8e3aa2b84fad52d98384cf47"
```
Comment (charlescng) [inline luajit-shopify.rb]: The checksum needs to be updated no?

Comment (ElvinEfendi) [reply luajit-shopify.rb]: good catch, it's updated now

File: luajit-shopify.rb [hunk 2]
```diff
@@ -1,8 +1,8 @@
 class LuajitShopify < Formula
   desc "Just-In-Time Compiler (JIT) for the Lua programming language"
   homepage "http://luajit.org/luajit.html"
-  url "http://luajit.org/download/LuaJIT-2.1.0-beta2.tar.gz"
-  sha256 "713924ca034b9d99c84a0b7b701794c359ffb54f8e3aa2b84fad52d98384cf47"
+  url "http://luajit.org/download/LuaJIT-2.1.0-beta3.tar.gz"
```
Comment (marc-barry) [inline luajit-shopify.rb]: Shouldn't we be using the same version we use in https://github.com/Shopify/luajit-build/pull/8? That would be https://github.com/openresty/luajit2/releases/tag/v2.1-20170808.

**Comment by ElvinEfendi** [comment]
https://github.com/Shopify/homebrew-shopify is currently being used only in dev and ci and we will drop it completely ASAP.

**Pull request closed without merging**